### DISCUSSION TITLE
Local setup/no domain setup

### DISCUSSION THREAD

**Discussion by alaricmiranda** [nextcloud/all-in-one#937] [Questions] [answered]
I wanted to try using this for my personal office but I plan to keep it local only. as such I'm currently stuck at the aio setup where it is asking me for a domain. can I not install this without setting up one ? as this is just a test run.

**Comment by szaimen** [comment, marked as answer]
This is not possible.

However you may have a look at this: https://github.com/nextcloud/all-in-one/issues/878
